dotcom-rendering/src/lib/braze/buildBrazeMessaging.ts

```diff
@@ -64,7 +65,7 @@ export const buildBrazeMessaging = async (
 		};
 	}
 
-	const isSignedIn = !!getCookie({ name: 'GU_U', shouldMemoize: true });
+	const isSignedIn = useIsSignedIn();
```

Comment:
we might need to pass `isSignedIn` as an argument to this function (`buildBrazeMessaging`) from where it is called, https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/useBraze.ts